.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,48 @@
+name: Close stale issues
+
+# Controls when the action will run.
+on:
+  schedule:
+  - cron: "0 6 * * *"
```

Comment:
It seems GitHub actions use UTC as a timezone, this means 10PM PDT.
Is this what you wanted?

Reply:
Yes, I just want to make it run at a regular time everyday and 10PM PDT is when our daily canary run.

Reply:
Good! 
Maybe add a comment like so:
`# Uses UTC so it runs at 10PM PDT`